python/plugins/db_manager/dlg_import_vector.py

```diff
@@ -25,7 +25,8 @@
 from builtins import range
 
 from qgis.PyQt.QtCore import Qt, QFileInfo
-from qgis.PyQt.QtWidgets import QDialog, QFileDialog, QMessageBox
+from qgis.PyQt.QtWidgets import QDialog, QFileDialog, QMessageBox, QApplication
+from qgis.PyQt.QtGui import QCursor
```

Comment:
What's this import for?

Reply:
Well i think we can drop it, it didn't add it personnaly, i think it's either from the build on QGIS or a missclick or something

Reply:
It's the result of a bad merge I think with the master branch. This has been removed 2 days ago in the master branch

Reply:
Oh okay ! Thanks . Anyway i will check all commit and confirm them if they are needed.